src/main/java/seedu/address/logic/commands/RedoCommand.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import seedu.address.logic.UndoRedoStack;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Redo the previously undone command.
+ */
+public class RedoCommand extends Command {
+
+    public static final String COMMAND_WORD = "redo";
+    public static final String MESSAGE_SUCCESS = "Redo success!";
+    public static final String MESSAGE_FAILURE = "no more commands to redo";
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        UndoRedoStack undoRedoStack = UndoRedoStack.getUndoRedoStack();
```

Comment:
I think you haven't implemented this method yet. Btw, I'm not sure if you can call a static method to get an instance of UndoRedoStack that isn't newly constructed. In AB4, logic manager keeps an instance of UndoRedoStack in LogicManager instead and passes that into Command.